mapactionpy_arcmap/arcmap_runner.py

```diff
@@ -41,34 +44,31 @@ def is_valid_directory(parser, arg):
         parser.error("The directory %s does not exist!" % arg)
         return False
 
+
 def getTemplate(orientation, cookbookFile, crashMoveFolder, productName):
-    gisFolder="GIS"
-    arcGisVersion="arcgis_10_6"
-    mappingDir="3_Mapping"
-    templateDirectoryName="32_MXD_Templates"
-    mapDirectoryName="33_MXD_Maps"
+    arcGisVersion = crashMoveFolder.arcgis_version
 
     # Need to get the theme from the recipe to get the path to the MXD
     cookbook = MapCookbook(cookbookFile)
     recipe = cookbook.products[productName]
 
-    templateDirectoryPath = os.path.join(crashMoveFolder, gisFolder, mappingDir, templateDirectoryName, arcGisVersion)
+    templateDirectoryPath = os.path.join(crashMoveFolder.path, crashMoveFolder.mxd_templates)
 
     if not(os.path.isdir(templateDirectoryPath)):
         print("Error: Could not find source template directory: " + templateDirectoryPath)
         print("Exiting.")
         sys.exit(1)
 
     if (recipe.category.lower() == "reference"):
-        templateFileName=arcGisVersion + "_" + recipe.category + "_" + orientation + "_bottom.mxd"
+        templateFileName = arcGisVersion + "_" + recipe.category + "_" + orientation + "_bottom.mxd"
     elif (recipe.category.lower() == "thematic"):
-        templateFileName=arcGisVersion + "_" + recipe.category + "_" + orientation + ".mxd"
+        templateFileName = arcGisVersion + "_" + recipe.category + "_" + orientation + ".mxd"
     else:
         print("Error: Could not get source MXD from: " + templateDirectoryPath)
         print("Exiting.")
         sys.exit(1)
 
-    mapDirectoryPath = os.path.join(crashMoveFolder, gisFolder, mappingDir, mapDirectoryName)
+    mapDirectoryPath = os.path.join(crashMoveFolder.path, crashMoveFolder.mxd_products)
 
     if not(os.path.isdir(mapDirectoryPath)):
```

Comment:
`crashMoveFolder.verify_paths()` would handle this and if the directory does not exist, it would generate error messages which are more consistant with the rest of the program.